src/modules/image.cpp

```diff
@@ -41,7 +40,15 @@ void waybar::modules::Image::refresh(int sig) {
 
 auto waybar::modules::Image::update() -> void {
   Glib::RefPtr<Gdk::Pixbuf> pixbuf;
-
+  if(config_["path"].isString())
+  {
+    path_ = config_["path"].asString();
+  }
+  else
```

Comment:
Please add if `config_["exec"].isString()` otherwise throw an error :)

Reply:
added else if statement to check exec is string